Resources/Prototypes/DeltaV/Entities/Structures/Windows/tinted_windows.yml

```diff
@@ -19,3 +19,25 @@
     state: tinted_window
   - type: Occluder
     boundingBox: "-0.5,-0.5,0.5,-0.3"
+
+- type: entity
+  id: WindowTintedReinforcedDirectional
+  parent: WindowReinforcedDirectional
+  name: directional reinforced tinted window
+  description: Don't smudge up the glass down there.
+  placement:
+    mode: SnapgridCenter
+    snap:
+    - Window
```

Comment:
inherited from the parent, remove

Reply:
its parented from the reinforced window, not from the other tinted window

Reply:
👍